geomesa-kafka/pom.xml

```diff
@@ -21,17 +21,11 @@
     </modules>
 
     <properties>
-        <kafka.guava.version>18.0</kafka.guava.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
 
     <dependencyManagement>
         <dependencies>
-            <dependency>
-                <groupId>com.google.guava</groupId>
-                <artifactId>guava</artifactId>
-                <version>${kafka.guava.version}</version>
-            </dependency>
```

Comment:
How did we remove the Kafka module's dependency on Guava?

Reply:
this may be one of those things we need to test haha. but i removed the use of guava loading caches and executors, so i don't think there are any direct dependencies on guava anymore (i don't think kafka uses the byte comparators i mentioned earlier).